srcpkgs/nagios/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'nagios'
+pkgname=nagios
+version=4.4.6
+revision=1
+build_style=gnu-configure
+
+_nagios_user="_nagios"
+_nagios_group="_nagios"
+_instdir="/usr/share/nagios"
+_bindir="/usr/bin"
+_vardir="/var/nagios"
+_confdir="/etc/nagios"
+_httpdconfdir="/etc/webapps/$pkgname/"
+_checkresultdir="/var/nagios/spool/checkresults"
+_cgibindir="/usr/libexec/nagios/cgi"
+
+configure_args="
+	--with-nagios-user=$_nagios_user
+	--with-nagios-group=$_nagios_group
+	--prefix=$_instdir
+	--bindir=$_bindir
+	--localstatedir=$_vardir
+	--sysconfdir=$_confdir
+	--with-httpd-conf=$_httpdconfdir
+	--with-checkresultdir=$_checkresultdir
+	--with-cgibindir=$_cgibindir
+	--enable-embedded-perl"
+make_build_args="all"
+make_install_args="
+	prefix=$_instdir
+	BINDIR=$_bindir
+	LOGDIR=$_vardir
+	CFGDIR=$_confdir
+	HTTPD_CONF=$_httpdconfdir
+	CHECKRESULTDIR=$_checkresultdir
+	CGIDIR=$_cgibindir
+	install install-config"
```

Comment:
We use spaces for indenting multiline strings.

Reply:
Does it need to be passed things like `bindir` in the configure script and in the make install step? Also, `bindir, prefix, localstatedir, sysconfdir` are already passed by the build style. Does it need `sysconfdir`, for example, to be the final directory?

Reply:
I had 2 spaces instead of tabs for multiline strings originally but it failed `xlint`:

(with spaces)

```
$ xlint srcpkgs/nagios/template 
srcpkgs/nagios/template:18: indent with tabs
```

Is this safe to ignore?

---

That's a good question - I'll do some testing and figure out if all of the variables are needed for both `configure` and `make`.

Reply:
Functions should be indented with tabs, multiline strings with a single space. So

```
a="1
 2"
```

We probably have to improve that xlint error.

Reply:
Gotcha - will update.